ext/hook/uhook.c

```diff
@@ -862,7 +917,7 @@ ZEND_METHOD(DDTrace_HookData, suppressCall) {
     RETURN_TRUE;
```

Comment:
Can we please log when disableJitInlining was not called before, to avoid forgetting calling it?